openedx_tagging/core/tagging/rest_api/v1/pagination.py

```diff
@@ -0,0 +1,10 @@
+"""
+Tagging API Pagination
+"""
+from rest_framework.pagination import PageNumberPagination
+
+
+class TaggingPagination(PageNumberPagination):
+    page_size_query_param = "page_size"
+    page_size = 100
+    max_page_size = 1000
```

Comment:
I believe you should just use [`edx_rest_framework_extensions.paginators.DefaultPagination`](https://github.com/openedx/edx-drf-extensions/blob/aa649e9ab13273a0624bfb623d4c80112631b601/edx_rest_framework_extensions/paginators.py#L9) instead. 

The platform [sets it as a global default for all APIs](https://github.com/openedx/edx-platform/blob/e0cee84ca9b4f9810583bacf4d7bb6ee4e4f427e/lms/envs/common.py#L3289) - maybe we can do so here too?

Reply:
Hi @bradenmacdonald!

> I believe you should just use [`edx_rest_framework_extensions.paginators.DefaultPagination`](https://github.com/openedx/edx-drf-extensions/blob/aa649e9ab13273a0624bfb623d4c80112631b601/edx_rest_framework_extensions/paginators.py#L9) instead.

In a discussion with @pomegranited https://github.com/open-craft/openedx-learning/pull/4#discussion_r1260712374, we talked that we shouldn't import `edx-platform` here. I think I misunderstood it and avoided importing other apps from edx in general. If this is not a problem, we should use this default pagination class.

> The platform [sets it as a global default for all APIs](https://github.com/openedx/edx-platform/blob/e0cee84ca9b4f9810583bacf4d7bb6ee4e4f427e/lms/envs/common.py#L3289) - maybe we can do so here too?

I think we can. Note that we have another rest_api in this app that will also be affected (not implemented, thou):

https://github.com/openedx/openedx-learning/blob/fb1fd0860f4873aae9250901a6af75a5f86384cc/projects/urls.py#L11

If this is fine, we can set a default for all APIs.

Reply:
Awesome, thanks for that recommendation @bradenmacdonald , I didn't know that existed.

@robrap  Yes please add this dependency and use [edx_rest_framework_extensions.paginators.DefaultPagination](https://github.com/openedx/edx-drf-extensions/blob/aa649e9ab13273a0624bfb623d4c80112631b601/edx_rest_framework_extensions/paginators.py#L9) instead of creating our own class here.

It's ok to just use it directly, and keep its `page_size` and `max_page_size` too -- we can adjust these later if we need them in the Tagging UI.

Reply:
@pomegranited I think you pinged the wrong person ;)

Reply:
Oops! Sorry for the noise @robrap!